index.html

```diff
@@ -42,7 +42,159 @@ <h1>I'm Denzel Glad to see you!</h1>
                 <span><a href="#"><i class="fab fa-angellist"></i></a></span>
             </div>
         </section>
```

Comment:
I know you did a good effort doing this project, but there are quite details that you should fix for font-weight, colors, padding/margin, images used, and so on, please do the best of yourself to replicate more the Figma design you selected 😉

- [ ] Fix styles used for the Figma design you selected.

![image](https://user-images.githubusercontent.com/60164239/129413884-b452518f-96c6-4d3a-9d54-4494ee4b8dd7.png)